### ISSUE TITLE
Ameliorated by flexion

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10123]
**Preferred term label:**
Ameliorated by flexion

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Obturator hernia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000050
- label: Ameliorated by flexion
- parents: [Ameliorated by (HP:0025254)](https://hpo.jax.org/app/browse/term/HP:0025254)
- pmid: 
- def: Applies to a sign or symptom that is improved or made more bearable by bending (flexing) a joint.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10123
- synonyms:
